packages/@react-spectrum/table/src/TableView.tsx

```diff
@@ -551,33 +578,68 @@ function ResizableTableColumnHeader(props) {
     ];
     return options;
   }, [allowsSorting]);
-  // if we're resizing another column, then hover shouldn't show the resizer of a different column
-  let showResizer = (isHovered && !columnState.currentlyResizingColumn) || columnState.currentlyResizingColumn === column.key;
 
   useEffect(() => {
     if (columnState.currentlyResizingColumn === column.key) {
-      focusSafely(ref.current);
+      // focusSafely won't actually focus because the focus moves from the menuitem to the body during the after transition wait
+      resizingRef.current.focus();
```

Comment:
Maybe linked to the weird `The resize handles don’t appear after tapping the “Resize column” menu option in Android in storybook iframe` behavior I noted. I can try poking around

Reply:
Looks like the issue goes away if I wrap this focus in a setTimeout,

Reply:
hahaha, but in a RAF it was failing for me